hq/test/schedule/unrecognised/IamUnrecognisedUsersTest.scala

```diff
@@ -0,0 +1,125 @@
+package schedule.unrecognised
+
+import com.gu.janus
+import com.gu.janus.model.{ACL, JanusData, SupportACL}
+import model._
+import org.joda.time.{DateTime, Seconds}
+import org.scalatest.{FreeSpec, Matchers}
+import schedule.unrecognised.IamUnrecognisedUsers.{USERNAME_TAG_KEY, getCredsReportDisplayForAccount, getJanusUsernames, unrecognisedUsersForAllowedAccounts}
+import utils.attempt.{FailedAttempt, Failure}
+
+class IamUnrecognisedUsersTest extends FreeSpec with Matchers {
+  val humanUser1 = HumanUser("ade.bimbola", true, AccessKey(NoKey, None), AccessKey(NoKey, None), Green, None, None, List(Tag(USERNAME_TAG_KEY, "ade.bimbola")))
+  val humanUser2 = humanUser1.copy(username = "john.akindele", tags = List(Tag(USERNAME_TAG_KEY, "john.akindele")))
+  val humanUser3 = humanUser1.copy(username = "khadija.omodara", tags = List(Tag(USERNAME_TAG_KEY, "khadija.omodara")))
+  val humanUser4 = humanUser1.copy(username = "nneka.obi", tags = List(Tag(USERNAME_TAG_KEY, "nneka.obi")))
+  val humanUser5 = humanUser1.copy(username = "john.doe", tags = List())
+  val humanUser6 = humanUser1.copy(username = "jane.doe", tags = List(Tag(USERNAME_TAG_KEY, "jane.doe")))
+  val allJanusUsernames = List("ade.bimbola", "john.akindele", "khadija.omodara", "nneka.obi", "jane.doe")
+
+  "findUnrecognisedIamUsers" - {
+    "get janus usernames" in {
```

Comment:
I don't think you've actually added this one (it's copied from elsewhere) but this test label doesn't appear to be doing a great job of describing what's being checked. Happy to leave this for now and come back to it later, or perhaps we can add some clarity to the label now?